.travis.yml

```diff
@@ -6,7 +6,9 @@ services:
   - docker
 
 script:
- - docker-compose build tests
+ - docker-compose build dev
+ - docker network create connection_api-tests
+ - docker-compose run --rm dev composer ci
```

Comment:
JFYI https://keboola.slack.com/archives/C9FEAHW7N/p1561376659016400

Reply:
přejmenoal jsem to na `app`